docs/environments.md

```diff
@@ -1,18 +1,17 @@
 # Environments
 
 ⚠️ URLs to be updated!
-
 We have several environments where the app can be deployed:
 
 
 |Env|URL|Purpose|How it's deployed|Backend env|
 |---|---|---|---|---|
 |local|http://localhost:3000/app|local development|`yarn start`|staging|
-|PRs   |`https://pr<PR_NUMBER>--safereact.review-safe.gnosisdev.com/app`|peer review & feature QA|for all PRs on push|staging|
-|release PRs|`https://pr<PR_NUMBER>--safereact.review-safe.gnosisdev.com/app`|pre-release regression testing|in PRs to `main`|**production** (sic!)|
-|dev  |https://safe-team.dev.gnosisdev.com/app/|preview of all WIP features|on push to the `dev` branch|staging|
-|staging|https://safe-team.staging.gnosisdev.com/app/|preview of features before a release|on push to `main`|staging|
-|production|https://gnosis-safe.io/app/|live app|deployed by DevOps (see the [Release Procedure](https://github.com/safe-global/safe-react/blob/ee39ef932d58e67db96756577814c1bfaf396d75/docs/release-procedure.md))|**production**|
+|PRs   |`https://<PR_NAME>--webcore.review-web-core.5afe.dev/`|peer review & feature QA|for all PRs on push|staging|
+|release PRs|`https://<PR_NAME>--webcore.review-web-core.5afe.dev/`|pre-release regression testing|in PRs to `main`|**production** (sic!)|
```

Comment:
This is not accurate anymore, btw. Can be deleted. RCs are just like regular branches.